### PR TITLE
Feature - HaNguyen - Basic Swift - Tin hoc dai cuong

### PR DESCRIPTION
**Pull request by at-hanguyen3**

### CODE REVIEW

File: BaiTap01/HaNguyenH/Bai12_Dieukien.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+//var dai: Int = 1
```
Comment (at-bienle) [inline BaiTap01/HaNguyenH/Bai12_Dieukien.playground/Contents.swift]: Delete unused code

File: BaiTap01/HaNguyenH/Bai2.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+var str = "Hello, playground"
+
+var x: Float = 2
+var y: Float = 3
+var kq: Float = 0
+func tu(_ x: Float, _ y: Float)-> Float {
```
Comment (at-bienle) [inline BaiTap01/HaNguyenH/Bai2.playground/Contents.swift]: Please add space before `->`, check it in another line

File: BaiTap01/HaNguyenH/Bai2.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    let tu: Float = Float(log(abs(x * x * y) / log(5) + atan(x + y)))
+    return tu
+}
+
+func mau(_ x: Float, _ y: Float)-> Float {
+    let mau: Float = exp(x) + cos(x + y)
+    return mau
+}
+
+kq = tu(x, y) / mau(x , y)
```
Comment (at-bienle) [inline BaiTap01/HaNguyenH/Bai2.playground/Contents.swift]: Should check `mau != 0`?

File: BaiTap01/HaNguyenH/Bai2_Dieukien.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+enum Result {
+    case phuongtrinhconghiemkep(Double, Double )
+    case phuongtrinhvonghiem
+    case phuongtrinh2nghiemthuc(Double, Double)
```
Comment (at-bienle) [inline BaiTap01/HaNguyenH/Bai2_Dieukien.playground/Contents.swift]: Should check case `phuongtrinhvosonghiem`?

File: BaiTap01/HaNguyenH/Bai2_Dieukien.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+}
+func bai2(a: Double, b: Double, c: Double, d: Double)-> Result{
```
Comment (at-bienle) [inline BaiTap01/HaNguyenH/Bai2_Dieukien.playground/Contents.swift]: Should show requirement (input, output)

Comment (at-hanguyen3) [reply BaiTap01/HaNguyenH/Bai2_Dieukien.playground/Contents.swift]: Nghĩa là sao a

**Pull request merged**